vsintegration/src/FSharp.Editor/Hints/InlineTypeHints.fs

```diff
@@ -44,8 +45,20 @@ module InlineTypeHints =
         (symbol: FSharpMemberOrFunctionOrValue)
         (symbolUse: FSharpSymbolUse) =
 
+        let isOptionalParameter = 
+            symbolUse.IsFromDefinition
+            && symbol.FullType.IsAbbreviation 
+            && symbol.FullType.TypeDefinition.DisplayName = "option"
+
+        let adjustedRangeStart =
+            if isOptionalParameter then 
+                // we need the position to start at the '?' symbol
+                mkPos symbolUse.Range.StartLine (symbolUse.Range.StartColumn - 1) 
```

Comment:
I wonder if there is a safer way to locate the range start.
Reason is, the syntax seems to allow a little more craziness, both with whitespace as well as line breaks.

```
    type MyType() =
        member _.MyMethod(?  beep: int, ?
                bap: int, ?boop: int) = ()
```


That being said, I still approve it because it fixes in the standard case and the worst thing that happens is that a formatting going against the style guide get's more type hints => survive-able in my opinion.

Reply:
Agree. There should be a way to get the argument start position using the existing symbol info. I'll take another look and see if we can capture cases with extraneous whitespace.